ipasetup.py.in

```diff
@@ -19,20 +19,21 @@ import sys
 
 
 PACKAGE_VERSION = {
-    'cryptography': 'cryptography >= 0.9',
+    'cryptography': 'cryptography >= 1.5',
     'dnspython': 'dnspython >= 1.11.1',
-    'gssapi': 'gssapi > 1.1.2',
+    'gssapi': 'gssapi >= 1.2.0',
```

Comment:
By the way this is yet another bike-shedding discussion. Unless the version is pinned or limited, pip will automatically download the **latest** non-pre build version from PyPI anyway. Any ```>=``` or ```>``` is only relevant for updates in an existing virtual env.